### ISSUE TITLE
238242882 A question about the Attic (Eric D)

### ISSUE THREAD

**Issue by ghost** [FreeUKGen/FreeUKRegProductIssues#1483]
Issue reported by **REGManager** at 2022-07-20 10:34:42 UTC
Time: 2022-07-20T10:31:50+00:00
Session ID: 485e5e207116ad9075a4095c8e601d40
Problem Page URL: [/attic_files/select_userid](/attic_files/select_userid)
Previous Page URL: [https://www.freereg.org.uk/manage_resources/new?locale=en](https://www.freereg.org.uk/manage_resources/new?locale=en)
Reported Issue:
When a file is replaced a copy of the previous version is saved in the Attic for reference.
But, if an entry is Edited online, is a version of the Batch saved in the Attic?  If not, we have no record of the previous version of that entry.
Eric    

![Screenshot](https://www.freereg.org.uk)

**Comment by FreecenBren** [comment]
Could someone please unsubscribe me from this list as I have retired

Tried other options. Cheers Brenda

**Comment by PatReynolds** [comment]
@DeniseColbert to send Brenda the link to unsubscribe.

**Comment by PatReynolds** [comment]
We may need to change the help.  Vino to take a look.

**Comment by DeniseColbert** [comment]
@Vino-S to test on test 3 if we replace attic files after online edit and let Eric know

**Comment by stsccfr** [comment]
See also #1493.  If a file is edited online and a new record is added, then this triggers a backup to the Attic, but if a record is deleted, or changed in some way, then this does not trigger a backup.

**Comment by Gev1990123** [comment]
Opened Pull Request: https://github.com/FreeUKGen/MyopicVicar/pull/2828

Testing to follow.

**Comment by stsccfr** [comment]
Yes, this is working now.  I tested this using STSCC/STSCMBBA.CSV on test3.  I tested deleting a record, adding a new record, editing an existing record, editing just the header, replacing the file, and deleting the file.  In all cases, backups were made to STSCC/.attic/  (I should mention for the sake of future testing that in order to trigger the backup to the .attic in the user's directory, one must download the file, except of course in the case that the file itself is deleted, in which case the backup is immediate.).  Marking Ready for Deployment.

**Comment by Vino-S** [comment]
Deployed
